packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -115,8 +115,10 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
     /// @notice Semantic version.
-    /// @custom:semver 2.2.0
-    string public constant version = "2.2.0";
+    /// @custom:semver 2.3.0
+    function version() public pure virtual returns (string memory) {
+        return "2.3.0";
```

Comment:
I think its better if we don't update the version as part of this PR given that I am trying to track a changelog for contracts in https://github.com/ethereum-optimism/optimism/issues/10592 and its going to make it more complicated